tests/units/classes/mock/generator.php

```diff
@@ -36,6 +36,21 @@ public function testSetAdapter()
         ;
     }
 
+    public function testSetParameterAnalyzer()
```

Comment:
@jubianchi  if there is a file that you can check, do this one. For me it's ok, but honestly I'm not sure I'm able to validate it